notebooks/Manage Map Areas with Group and Index.ipynb

```diff
@@ -0,0 +1,456 @@
+    "            # get items url\n",
+    "            basemap_item = gis.content.get(basemap[\"itemId\"])\n",
+    "            basemap_url = basemap_item[\"url\"]\n",
+    "\n",
+    "        elif \"styleUrl\" in basemap:\n",
+    "            # get source url from \n",
+    "            vtl = VectorTileLayer(basemap.styleUrl,gis)\n",
+    "            basemap_url = VectorTileLayer(vtl.properties.sources.esri.url,gis).url\n",
+    "\n",
+    "        if basemap_url is not None:\n",
+    "            # using the attribute values for min_scale and max_scale - loop through service supported levels and include\n",
+    "            \n",
+    "            try:\n",
+    "                basemap_levels_array = []\n",
+    "                map_service = MapImageLayer(basemap_url,gis)\n",
+    "                for lod in map_service.properties.tileInfo[\"lods\"]:\n",
+    "                    # min_scale and max_scale are required\n",
+    "                    if map_area.get_value(\"min_scale\") >= lod[\"scale\"] >= map_area.get_value(\"max_scale\"):\n",
+    "                        # include this level in basemap_levels\n",
+    "                        basemap_levels_array.append(str(lod[\"level\"]))\n",
+    "                        \n",
+    "            except Exception as e:  \n",
+    "                print(\"  failed - processing basemap levels\")\n",
+    "                print(e)\n",
+    "                \n",
+    "            basemap_levels = \",\".join(basemap_levels_array)\n",
+    "\n",
+    "            #set the web map tile service parameters url and levels\n",
+    "            map_area_tile_services.append({\"url\":basemap_url,\"levels\":basemap_levels})    \n",
+    "        \n",
+    "    return map_area_tile_services\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Processing each web map in group, as defined by index"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"-- begin processing map areas --\")\n",
+    "process_begin_time = datetime.datetime.now()       \n",
+    "\n",
+    "# optional - truncate the logs - this can be removed to based on retention preferences\n",
+    "#map_areas_hfl.tables[0].manager.truncate()     \n",
+    "\n",
+    "# get all group shared items\n",
+    "for item in manage_map_areas_group.content():\n",
+    "    # if the shared item is not a web map skip it \n",
+    "    if( item[\"type\"] != \"Web Map\" ):\n",
+    "        continue # to next item in for loop   \n",
+    "\n",
+    "    print(\"--------------\")\n",
+    "    print(f\"processing web map: {item['title']}\")        \n",
+    "\n",
+    "    # Cast item to Map\n",
+    "    web_map = Map(item)\n",
+    "\n",
+    "    # create map area as defined in map_areas_hfl\n",
+    "    where_clause =  f\"webmap_id IN ('{item.id}', 'ALL') and status = 'Active'\"\n",
```

Comment:
Seems like this is essential for the user to input into their polygon. Could we write in the intro somewhere that this is required for the user to do? Or this is implied? Thank you!